Dockerfile

```diff
@@ -153,3 +151,9 @@ RUN chmod a+rX /srv/templates
 COPY resources/jupyter_notebook_config.partial.py /tmp/
 RUN cat /tmp/jupyter_notebook_config.partial.py >> /home/jovyan/.jupyter/jupyter_notebook_config.py && \
     rm /tmp/jupyter_notebook_config.partial.py
+
+# Ensure notebook, ipywidets are up to date
+RUN conda install --quiet --yes \
+    'notebook=4.2' \
+    'ipywidgets=5.1' \
```

Comment:
re: widgets amiss, the base images are doing the `jupyter nbextension enable --py widgetsnbextension`. If you're reinstalling here, wonder if that breaks things ... ?